app/views/request/_pictureseries_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="row">
+<div class="col-12 col-lg-8">
+    <%= form_with(url: solr_document_request_path,
+                  model: request,
+                  method: :post,
+                  data: {
+                    "form-validation-target": "form"
+                  }) do |f| %>
+      <%= hidden_field :request, :holdings_id %>
+      <%= hidden_field :request, :item_id %>
+
+      <div class="mb-3 row">
+        <%= f.label :enumeration, t("requesting.label.call_numbers"), class: "col-md-3 col-form-label text-md-right" %>
+        <div class="col-sm-6">
+          <%= f.text_field :enumeration, class: "form-control", maxlength: 80, data: {"form-validation-target": "required", action: "form-validation#validate"} %>
+        </div>
+        <span id="volume-help" class="form-text text-muted col-sm-3"><%= t("requesting.help.volume") %></span>
```

Comment:
should this be enumeration-help or call-number-help and requesting.help.enumeration. (also in posterseries_form)